plasmapy/particles/particle_class.py

```diff
@@ -292,7 +322,7 @@ class Particle(AbstractParticle):
     'D 0+'
 
     If the particle doesn't belong to one of those categories, then
-    these attributes return `None`.
+    these attributes provide `None`.
```

Comment:
Eh, provide... maybe...
```suggestion
    these attributes are `None`.
```